### ISSUE TITLE
Method name suggestions for version 1.0.0

### ISSUE THREAD

**Issue by arasatasaygin** [arasatasaygin/is.js#44]
Some method names are not as good as it should be. (Thanks to my poor English)
However I don't want to change api so fast. So please list your suggestions on method names here for now:

@tbassse suggested here https://github.com/arasatasaygin/is.js/issues/43 :
`is.under()` > `is.below()`

@colindresj suggested here https://github.com/arasatasaygin/is.js/issues/23:
`is.setNamespace()` > `is.noConflict()`

**Comment by pducks32** [comment]
You should use —> or something instead of > because that makes it look like `is.under()` is greater/better than `is.below()` and I was really confused for a minute.

**Comment by arasatasaygin** [comment]
:) edited

**Comment by Guru107** [comment]
Just a suggestion, If naming ```is.object()```  to  ```is.json()``` it gives more readability to the type. Or atleast providing an alias ```json``` would be great.

**Comment by njenia** [comment]
I would change `is.within(num, min, max)` to `is.between(num, min, max)`. I think that *between* is clearer.

**Comment by Guru107** [comment]
I would suggest to change the name of two functions ```is.function``` and ```is.null``` since they are JavaScript keywords. Something like ```is.typeFunction()``` and ```is.typeNull()``` or if someone can suggest some good and concise names that would be great.

**Comment by pducks32** [comment]
I agree the name isn't great, but I think it's tenseness is really straight forward and is kind of a theme of the library.

-Patrick Metcalfe

**Comment by paracycle** [comment]
I would like to see `is.all.xxx` being renamed to `is.every.xxx` because the first one isn't correct grammatically, it should be `are.all.xxx`. The second form works both grammatically and expresses intent clearly, IMHO.

**Comment by pducks32** [comment]
I disagree because in many other languages using all is standard so it maps nicely here. Plus it may not be grammatically correct but it is, I believe, more clear. I don’t know though; just my thought. [Python all](https://docs.python.org/2/library/functions.html <https://docs.python.org/2/library/functions.html>) and [Ruby's all?](http://ruby-doc.org/core-2.2.1/Enumerable.html#method-i-all-3F <http://ruby-doc.org/core-2.2.1/Enumerable.html#method-i-all-3F>)

**Comment by pducks32** [comment]
I disagree because in many other languages using all is standard so it maps nicely here. Plus it may not be grammatically correct but it is, I believe, more clear. I don’t know though; just my thought. [Python all](https://docs.python.org/2/library/functions.html) and [Ruby's all?](http://ruby-doc.org/core-2.2.1/Enumerable.html#method-i-all-3F)

**Comment by paracycle** [comment]
@pducks32 I am aware of those but none of those methods are trying to make grammatical sense. 

Besides I could counter with [Underscore.js every](http://underscorejs.org/#every) and [LoDash's every](https://lodash.com/docs#every) (which is also aliased as `all`) and, finally, the [Javascript every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every).

So it seems to be the way Javascript people like it.

**Comment by pducks32** [comment]
Actually I agree. All of those are Js libraries so I actually like the mapping better.
So yea I support `every` 👍

**Comment by daleeidd** [comment]
@Guru107 I would like to propose amending the suggestion from renaming to aliasing since using reserved words as property names is only invalid in ES3. https://kangax.github.io/compat-table/es5/
